test/functional/wallet_bumpfee.py

```diff
@@ -130,7 +130,7 @@ def test_invalid_parameters(self, rbf_node, peer_node, dest_address):
         assert_raises_rpc_error(-8, "Insufficient total fee 0.00000141", rbf_node.bumpfee, rbfid, fee_rate=INSUFFICIENT)
 
         self.log.info("Test invalid fee rate settings")
-        assert_raises_rpc_error(-4, "Specified or calculated fee 0.141 is too high (cannot be higher than -maxtxfee 0.10",
+        assert_raises_rpc_error(-4, "New fee rate 1.00 is too high (cannot be higher than -maxfeerate 0.10)",
```

Comment:
Because after https://github.com/bitcoin/bitcoin/commit/eadc3a4500d6b7be263a2981602c20f6e5eb13a4 we now check for `maxfeerate` threshold first before checking `maxtxfee` threshold.
And the specified fee rate is above default `maxferate`, as such it will be hit first.

But I agree, I will add another test where  specified bump fee rate is below the `maxfeerate` but we hit `maxtxfee` due to size or maybe just restart with `maxtxfee`  set to a lower value ?

Reply:
Either sounds fine! The main point is we shouldn't be removing test cases for maxtxfee while adding maxtxfeerate.

Reply:
Thanks updated the tests.